tests/integration.rs

```diff
@@ -6731,9 +6731,11 @@ mod common_parallel {
 }
 
 mod common_sequential {
+    #[allow(unused_imports)]
```

Comment:
Please use `#[cfg(not(feature = "mshv"))]` instead.

Reply:
Done!